pyaedt/generic/settings.py

```diff
@@ -185,8 +185,8 @@ def lsf_timeout(self, value):
 
     @property
     def aedt_version(self):
-        """AEDT version in the form ``"2023.x"``. If AEDT version is 2022 R2 and later,
-        the bounding box evaluation by exporting a SAT file is disabled."""
+        """AEDT version in the form ``"2023.x"``. In AEDT 2022 R2 and later,
+        bounding box evaluation by exporting a SAT file is disabled."""
```

Comment:
```suggestion
        evaluating a bounding box by exporting a SAT file is disabled."""
```
On rereading, "bounding box evaluation" seemed harder to comprehend.